app/services/metrc_service/plant/move.rb

```diff
@@ -55,17 +55,23 @@ def is_included?(previous_growth_phase, growth_phase) # rubocop:disable Naming/P
         GROWTH_CYCLES[previous_growth_phase.downcase.to_sym]&.include?(growth_phase.downcase.to_sym)
       end
 
+      def within_phases?(zone_name, expected_zones)
+        return false unless zone_name && expected_zones
+
+        expected_zones.any? { |phase| zone_name.include?(phase) }
+      end
+
       def next_step(previous_growth_phase = nil, new_growth_phase = nil) # rubocop:disable Metrics/CyclomaticComplexity,Metrics/PerceivedComplexity
         return DEFAULT_MOVE_STEP if previous_growth_phase.nil? || new_growth_phase.nil?
 
         new_growth_phase.downcase!
 
-        return :move_plant_batches if previous_growth_phase.include?('clone') && new_growth_phase.include?('clone')
-
-        return :move_plants if previous_growth_phase.include?('veg') && new_growth_phase.include?('veg')
+        return :move_plant_batches if within_phases?(previous_growth_phase, %w[veg clone]) && within_phases?(new_growth_phase, %w[veg clone])
```

Comment:
I might be wrong, but I think `previous_growth_phase` has a value such as `"Vegetative"`, as returned by `normalized_growth_phase`, right? but the `expected_zones` you're passing here are the short strings `"veg"` and `"clone"`. If the zone name is normalized into a long string, shouldn't we be using that type of strings for comparison and to avoid confusion?